File: Darts/src/burlton/dartzee/code/db/sanity/SanityCheckResultEntitiesSimple.java
package burlton.dartzee.code.db.sanity;

import burlton.dartzee.code.db.AbstractEntity;

import java.util.List;

public class SanityCheckResultEntitiesSimple extends AbstractSanityCheckResultEntities
{
	private String description = "";
	
	public SanityCheckResultEntitiesSimple(List<? extends AbstractEntity<?>> entities, String description)
	{
		super(entities);

		this.description = description;
	}

	@Override
	public String getDescription()
	{
		return description;
	}

}

File: Darts/src/burlton/dartzee/code/db/sanity/SanityCheckResultEntitiesSimple.kt
package burlton.dartzee.code.db.sanity

import burlton.dartzee.code.db.AbstractEntity

class SanityCheckResultEntitiesSimple(entities: List<AbstractEntity<*>>, val desc: String) : AbstractSanityCheckResultEntities(entities)
{
    override fun getDescription() = desc
}
